src/types_.py

```diff
@@ -37,12 +37,14 @@ class UserInputs(typing.NamedTuple):
             migration mode.
         github_access_token: A Personal Access Token(PAT) or access token with repository access.
             Required in migration mode.
+        base_branch: The branch the the documentation is stored on after updating discourse.
```

Comment:
`The branch where the documentation is stored` there's a double `the`

Reply:
Thanks, should be fixed shortly